hermes/probe/read/read_file.go

```diff
@@ -0,0 +1,80 @@
+// TODO (#70) add license header and author line
+// TODO (#69) add doc strings
+
+package read
+
+import (
+	"context"
+	"crypto/sha1"
+	"fmt"
+	"io"
+	"time"
+
+	"cloud.google.com/go/storage"
+	"github.com/google/cloudprober/logger"
+	"github.com/googleapis/google-cloud-go-testing/storage/stiface"
+	"github.com/googleinterns/step224-2020/hermes/probe"
+	m "github.com/googleinterns/step224-2020/hermes/probe/metrics"
+)
+
+func checkSumFromName(fileName string) (string, error) {
+	if len(fileName) < 11 {
+		return "", fmt.Errorf("The filename stored in the journal is shorter than 11 characters, hence does not match Hermes_id_checksum")
+	}
+	return fileName[10:len(fileName)], nil
```

Comment:
10 is a magic number
https://en.wikipedia.org/wiki/Magic_number_(programming)

Avoid magic numbers in general.
I would put the prefix in a string const then use `len()`.
Make sure that string const is referenced everywhere instead of hard-coding the string in-line.

Reply:
A good use of `len()` here would be: `len("Hermes_ID_"`).

Reply:
Even better would be to use the recently introduced FileNamePrefixLength variable.

Also note that there's no need to specify `len(fileName)`, just `return filenName[FileNamePrefxLength:], nil`.

Reply:
removed 10 added a const